sematic/plugins/building/docker_builder.py

```diff
@@ -15,11 +15,12 @@
 from typing import Any, Dict, List, Optional, Tuple
 
 # isort: off
-# there is a collision between the docker-py library and the sematic/docker directory
 
 # Third-party
 import docker
 import yaml
+
+# there is a collision between the docker-py library and the sematic/docker directory
```

Comment:
I'm unclear on the purpose of this comment

Reply:
Clarified.